docs/app/pages/components/sections/date-time-picker/time-picker/time-picker.component.html

```diff
@@ -0,0 +1,122 @@
+<timepicker [ngModel]="myTime" [min]="minTime" [max]="maxTime" [hourStep]="hstep" [minuteStep]="mstep" [showMeridian]="ismeridian"
```

Comment:
Yes please, you can verify your changes by checking for a horizontal scrollbar on the code snippet in the documentation.